sdk/compute/azure-mgmt-compute/setup.py

```diff
@@ -36,7 +36,10 @@
     pass
 
 # Version extraction inspired from 'requests'
-with open(os.path.join(package_folder_path, 'version.py'), 'r') as fd:
+# Version extraction inspired from 'requests'
```

Comment:
@zikalino  duplicated comment

Reply:
yeah, noticed, have to fix update_pr thingy :-(

Reply:
https://github.com/Azure/azure-sdk-for-python/pull/11039/files